homeassistant/components/flume/sensor.py

```diff
@@ -11,7 +11,7 @@
 from homeassistant.helpers.entity import Entity
 from homeassistant.util import Throttle
 
-_LOGGER = logging.getLogger(__name__)
+Logger = logging.getLogger(__name__)
```

Comment:
Please change this back to previous version.

Reply:
Resolved